### DISCUSSION TITLE
V1.1.1, I still don't care about cookies

### DISCUSSION THREAD

**Discussion by OhMyGuus** [OhMyGuus/I-Still-Dont-Care-About-Cookies#2265] [Announcements]
# I still don't care about cookies  V1.1.1

## (RECOMMENDED) Installation using official chrome & Firefox store
- [**Firefox** browser add-ons](https://addons.mozilla.org/en-US/firefox/addon/istilldontcareaboutcookies/
)
- [**Chrome** Web Store](https://chrome.google.com/webstore/detail/i-still-dont-care-about-c/edibdbjcniadpccecjdfdjjppcpchdlm)
- [**Edge** Add-ons](https://microsoftedge.microsoft.com/addons/detail/i-still-dont-care-about-/kkacdgacpkediooahopgcbdahlpipheh)

## Manual installation guide

- [Manual installation guide for Firefox](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/wiki/Firefox-installation-guide)
- [Manual installation guide for Chrome](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/wiki/Chrome-installation-guide)

## Contributors & website reporters
Thanks a lot to the contributors who are willing to contribute to the project by either reporting websites or by willing to implement websites into the extension 👍 
(Especially thanks a lot @appeasementPolitik for contributing a lot this release 😄 )

## Contact
_If u need to contact me create a GH issue or message me on discord 
Discord: ThaGuus#2140_
Email: Info[AT]istilldontcareaboutcookies.com
Github: [Issues](https://github.com/OhMyGuus/I-Dont-Care-About-Cookies/issues/new)

## What's Changed
* Generate new commons and rules with ruleclean and merge some commons by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/770
* Websites fix by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/893
* Update Simplified Chinese Translation by @CJYKK in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/948
* Fix for reported websites #15 by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/983
* i18n AMO links by @Gitoffthelawn in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/1111
* Clean rules by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/1090
* Fix for reported websites #16 by @appeasementPolitik in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/1268
* New Crowdin translations by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/437
* Fix for reported websites #14 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/939
* WIP support for reported websites  by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/1877
* Update V1.1.1 by @OhMyGuus in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/2264

## New Contributors
* @Gitoffthelawn made their first contribution in https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/pull/1111

**Full Changelog**: https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/compare/v1.1.0...v1.1.1

<hr /><em>This discussion was created from the release <a href='https://github.com/OhMyGuus/I-Still-Dont-Care-About-Cookies/releases/tag/v1.1.1'>V1.1.1, I still don't care about cookies</a>.</em>

**Comment by rcalixte** [comment]
@OhMyGuus It appears that the extension's .crx file is not available for download from the assets?

**Reply by OhMyGuus** [reply]
Good one, I indeed forgot to add it 👍 Added it now to the release 
(removed the xpi since it was still v1.1.0 and ff requires review before I can download the xpi)

**Reply by rcalixte** [reply]
Thank you for everything!
